src/EditorFeatures/CSharpTest/Diagnostics/UseAutoProperty/UseAutoPropertyTests.cs

```diff
@@ -1217,6 +1217,54 @@ int P
 @"class Class
 {
     int P { get; set; } = 1;
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseAutoProperty)]
+        [WorkItem(25401, "https://github.com/dotnet/roslyn/issues/25401")]
+        public async Task TestMissingIfAccessibilitiesDiffer1()
+        {
+            await TestMissingInRegularAndScriptAsync(
```

Comment:
This should be fixed as 

public int P { protected get; set; }

in C#.

Reply:
yeah... I don't see a reason to disallow it here